### ISSUE TITLE
[Feature Request] Attaching files to define task, bootstrap org and name, and for including reference files.

### ISSUE THREAD

**Issue by cryptictech** [OpenBMB/ChatDev#321]
It would be nice to define a task file for defining a longer specified task as doing so from the command line can make the command too long and hard to understand. I'm sure this is possible with a Linux command, although I'm worried that it could also be hard to read, I'm thinking perhaps through a `--tasks-file` option.

Adding a separate file for org and project name or even pulling from an environment file/variable could help shorten or bootstrap the run command, using the parameter option over the defined variable to allow on the fly modifications without having to modify the file.

Lastly it would be nice to add additional files that the program can use as documentation or for reference, this could be through a single option like `--reference-file` or through different options to specify different file uses, for instance `--doc-file` or `--requirements-file`.

**Comment by Liagim01** [comment]
Thanks for the great suggestion, the task file and separation of different files are great ideas

**Comment by aaronjolson** [comment]
Yeah, the gpt-engineer project uses the prompt file, which handles free form text. It is much easier to specify complex requirements using that interface. I often paste large code blocks or entire files into that prompt file. 
If I was to flatten out that code in order to make it a one line string that can be input via command line arguments, it loses a lot of important form and meaning.

**Comment by Linuxscot1974** [comment]
I was able to update the code to allow for file inclusion.
I have not had a chance to see if it will run, if someone can check it out I would appreciate it.
i have linked it to my github [repo](https://github.com/Linuxscot1974/chatdev_updates/blob/main/)
